tests/scripts/task_python_unittest.sh

```diff
@@ -31,9 +32,9 @@ if [ -z "${TVM_UNITTEST_TESTSUITE_NAME:-}" ]; then
 fi
 
 # First run minimal test on both ctypes and cython.
-run_pytest ctypes ${TVM_UNITTEST_TESTSUITE_NAME}-platform-minimal-test tests/python/all-platform-minimal-test
-run_pytest cython ${TVM_UNITTEST_TESTSUITE_NAME}-platform-minimal-test tests/python/all-platform-minimal-test
+run_pytest --parallel ctypes ${TVM_UNITTEST_TESTSUITE_NAME}-platform-minimal-test tests/python/all-platform-minimal-test
+run_pytest --parallel cython ${TVM_UNITTEST_TESTSUITE_NAME}-platform-minimal-test tests/python/all-platform-minimal-test
 
 # Then run all unittests on both ctypes and cython.
-run_pytest ctypes ${TVM_UNITTEST_TESTSUITE_NAME} tests/python/unittest
-run_pytest cython ${TVM_UNITTEST_TESTSUITE_NAME} tests/python/unittest
+run_pytest --parallel ctypes ${TVM_UNITTEST_TESTSUITE_NAME} tests/python/unittest
+run_pytest --parallel cython ${TVM_UNITTEST_TESTSUITE_NAME} tests/python/unittest
```

Comment:
Sorry for being late reviewing this PR. I was thinking whether we could make this `--parallel` here to be disabled somehow. I think it is OK and expected that it is enabled by default, but to make this scripts reusable, it would be great to have a way to disable things. Perhaps an environment variable is enough?

An idea would be standardising `$PYTEST_PARALLEL=ON|OFF ? YES|NO ? 1|0?` as the way to control that in all scripts, and document that.

(same comment applies to task_python_vta_fsim.sh and other in this same PR)

Reply:
Yes, that works! I missed it from the scripts. Can we document that somewhere?